simple_example_MackeyGlass.py

```diff
@@ -154,12 +154,12 @@ def set_seed(seed=None):
 
 
 internal_trained = reservoir.train(inputs=[train_in,], teachers=[train_out,], wash_nr_time_step=initLen, verbose=False)
-output_pred, internal_pred = reservoir.run(inputs=[test_in,], reset_state=False)
+output_pred, internal_pred = reservoir.run(inputs=[test_in,], reset_state=True)
```

Comment:
In this line, could you put back the argument "True" to "False" please?
`output_pred, internal_pred = reservoir.run(inputs=[test_in,], reset_state=True)`